magicctapipe/scripts/lst1_magic/lst1_magic_create_irf.py

```diff
@@ -2,17 +2,34 @@
 # coding: utf-8
 
 """
-This script creates the IRFs with input MC DL2 data. Now it can create only
-point-like IRFs (effective area, energy migration and background hdus_irf).
-If input data is only gamma MC, it skips the creation of a background HDU.
+This script processes MC DL2 events and creates the IRFs. Now it can
+create only point-like IRFs, i.e., the effective area, energy migration
+and background model. If the input data is only gamma MC, it also skips
+the creation of the background model.
+
+There are 4 different IRF types which can be specified by the "irf_type"
+setting in the configuration file. The "hardware" type is supposed for
+the hardware trigger between LST-1 and MAGIC, allowing for the events of
+all the telescope combinations. The "software(_only_3tel)" types are
+supposed for the software coincidence with LST-mono and MAGIC-stereo,
+allowing for only the events triggering both M1 and M2. The "software"
```

Comment:
maybe best to mention that MAGIC only events without LST1 are excluded for this "software" ones (unless we want to change it)

Reply:
Yes, this is already mentioned in the next line, but I slightly modified it.